src/app/components/graph/graph-launcher.html

```diff
@@ -192,7 +192,7 @@ <h6>
                                            class="field-input form-control input-dark"
                                            ng-model="selectorService.selection.dirty.include"
                                            placeholder="..." />
-                                    <div class="field-label">--models</div>
+                                    <div class="field-label">--select</div>
```

Comment:
note: this changes the `--models` label to `--select`, which feels appropriate if not 100% accurate all of the time. Probably something we should give more thought to for a future iteration of this UI

Reply:
I agree with this change. It feels right to have the DAG mirror the behavior and syntax of `dbt ls`. And as Jake pointed out to me (not long ago), `dbt ls --models ...` is basically `dbt ls --resource-type model --select`.